pkg/parcacol/normalizer.go

```diff
@@ -321,10 +321,11 @@ func (n *MetastoreNormalizer) NormalizeLocations(
 		}
 
 		req.Locations = append(req.Locations, &pb.Location{
-			Address:   addr,
-			IsFolded:  location.IsFolded,
-			MappingId: mappingId,
-			Lines:     lines,
+			Address:      addr,
+			IsRawAddress: false,
```

Comment:
We can set `IsRawAddress: !n.isAddrNormEnabled` once https://github.com/parca-dev/parca/pull/2955 is merged.